Typescript error: "Could not find a declaration file for module X implicitly has an 'any' type."

```
error TS7016: Could not find a declaration file for module '@glokon/guacamole-common-js'. '/project/node_modules/@glokon/guacamole-common-js/lib/esm/guacamole.js' implicitly has an 'any' type.
  There are types at '/project/node_modules/@glokon/guacamole-common-js/guacamole.d.ts', but this result could not be resolved when respecting package.json "exports". The '@glokon/guacamole-common-js' library may need to update its package.json or typings.
```

This is using Typescript 5.5.4

--- a/package.json
+++ b/package.json
@@ -7,8 +7,14 @@
   "module": "lib/esm/guacamole.js",
   "exports": {
     ".": {
-      "import": "./lib/esm/guacamole.js",
-      "require": "./lib/cjs/guacamole.js"
+      "import": {
+        "types": "./guacamole.d.ts",
+        "default": "./lib/esm/guacamole.js"
+      },
+      "require": {
+        "types": "./guacamole.d.ts",
+        "default": "./lib/cjs/guacamole.js"
+      }
     }
   },
   "types": "guacamole.d.ts",

